frontend/routes/dashboard/dashboard.component.tsx

```diff
@@ -96,10 +96,7 @@ export class Dashboard extends React.PureComponent<IProps, any> {
 
 	public renderDashboardRoute = ({match}) => (
 		<Content>
-			{ !this.props.isInitialized && this.props.isPending
```

Comment:
Is it correct?

Reply:
Yes, we are moving from having most of the data in the /teamspace.json , to several endpoints with less payload. That means that each tab will have  the responsibility of knowing which part of the state it needs and load it accordingly, that also means the loading state for each tab varies.

Reply:
Awesome!